src/test/java/hw5/steps/ThenSteps.java

```diff
@@ -0,0 +1,112 @@
+package hw5.steps;
+
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static hw5.utils.Utils.getNotEmptyTextFromWebElements;
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class ThenSteps extends AbstractStep {
```

Comment:
ctrl + l - to format file. Pls make it

Reply:
Hm... But file is formatted on my local computer. Strange things go on here, you know...